bambi/models.py

```diff
@@ -208,6 +208,11 @@ def fit(
             using the ``fit`` function.
             Finally, ``"laplace"``, in which case a Laplace approximation is used and is not
             recommended other than for pedagogical use.
+            To use the PyMC numpyro and blackjax samplers, use ``mcmc-numpyro`` or ``mcmc-blackjax``
+            respectively. Both methods will only work if you can use NUTS sampling, so your model must
+            be differentiable. It is also recommended, if using ``mcmc-numpyro`` or
+            ``mcmc-blackjax``, to set the kwarg of ``chain_method`` to either ``parallel`` or
+            ``vectorized`` for optimal performance.
```

Comment:
```suggestion
            To use the numpyro and blackjax samplers, use ``nuts_numpyro`` or ``nuts_blackjax``
            respectively. Both methods will only work for differentiable models. If using ``nuts_numpyro`` or
            ``nuts_blackjax``, it is recomended to set the kwarg ``chain_method`` to either ``parallel`` or
            ``vectorized`` for optimal performance.
```

Reply:
Not clear to me what users should do with the chain_method argument. How should they pick between ``vectorized`` or ``parallel``?

Reply:
I can add some explanation if you like here if it won't make the docs a bit too verbose? It's whether they use pmap or vmap in Jax (https://jax.readthedocs.io/en/latest/jax-101/06-parallelism.html). It depends a lot on whether you are running on a GPU or CPU, how many GPUs you have etc... maybe better for users to experiment themselves?

Reply:
The default of 'sequential' is generally slower in all cases I can think of - both CPUs and GPUs can benefit from vectorisation and parallelisation (unless you're on a one core machine from a few decades ago!) it's kind of a crazy default I think.

Reply:
We can have some explanation as part of the notebook example. I made another suggestion trying to provide some explanation but keeping concise, feel free to improve it. Notice that in PyMC the default is parallel